src/main/java/com/google/sps/servlets/Search.java

```diff
@@ -22,51 +22,46 @@
 @WebServlet("/search")
 public class Search extends HttpServlet {
       DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-
+    
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        System.out.println("This is get request");
-    }
 
-    
-    @Override
-    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        //CASE 1: RESPONSE RETURNED AS TITLE CATEGORY
         System.out.println("this is post request");
-        // Enumeration enumeration = request.getParameterNames();
-        // Map<String, Object> modelMap = new HashMap<>();
-        // while(enumeration.hasMoreElements()){
-        //     String parameterName = (String) enumeration.nextElement();
-        //     modelMap.put(parameterName, request.getParameter(parameterName));
-        // }
-        // JSONObject json = new JSONObject();
-        // for (Map.Entry<String, Object> entry : modelMap.entrySet()) {
-        //     json.put(entry.getKey(), entry.getValue());
-        // }
-        // response.setContentType("application/json");
-        // response.getWriter().println(json.toString());
-       
-        final String questionTitle = "question";
-        String questionTitleValue = request.getParameter("questionTitle");
+        String returnedValue = request.getParameter("searchBar");
         final String entity_questionDB = "survey";
-        Query query = new Query(entity_questionDB).addFilter(questionTitle,FilterOperator.EQUAL,questionTitleValue);
+        System.out.println("Returned value is " + returnedValue);
+
+
+        //If is question TITLE
+        final String questionTitle = "question";
+        Query query = new Query(entity_questionDB).addFilter(questionTitle,FilterOperator.EQUAL,returnedValue);
```

Comment:
are we changing the use of EQUALS for matching?

Reply:
I modified to IN, which it will basically use the same procedure of EQUAL but in an list of words. I've updated on the latest commit